gnomad_qc/v3/analysis/vary_subpop_params.py

```diff
@@ -0,0 +1,215 @@
+import argparse
+import hail as hl
```

Comment:
very minor comment, but I think we typically have hail (and I think pandas) imports after argparse and logging. Then gnomad methods then gnomad QC. No functionality change, just formatting.